### DISCUSSION TITLE
Asus TUF A16 No Mode on GPU in G Helper

### DISCUSSION THREAD

**Discussion by Keng0818** [seerge/g-helper#3427] [Q&A]
Hi! A newbie here. First time buying a Asus TUF laptop and totally new to the environment and all. I've bumped into this G-Helper as Armoury Crate is confusing. I just want to ask is it really no mode/s in my GPU settings in G-Helper? 

Thanks a lot for your answers, really appreciate it!

![Untitled](https://github.com/user-attachments/assets/800e41c7-f787-4202-acce-14639fcbbf9e)

**Comment by werty94** [comment]
Hi, 

The Balanced plan is recommended to use with G-Helper. Enable it and check the GPU section. There are additional settings in Fans+Power related to the GPU.

**Reply by Keng0818** [reply]
Hi,

Thanks for your reply. I've tried all of the modes but i can't still change the GPU Mode. I've seen on other videos that they can change their GPU Mode to Eco Balance (not really sure). How come i can't change mine?
